core/src/main/java/org/testcontainers/utility/RegistryAuthLocator.java

```diff
@@ -183,6 +183,10 @@ private AuthConfig authConfigUsingStore(final JsonNode config, final String repo
         final JsonNode credsStoreNode = config.get("credsStore");
         if (credsStoreNode != null && !credsStoreNode.isMissingNode() && credsStoreNode.isTextual()) {
             final String credsStore = credsStoreNode.asText();
+            if (isBlank(credsStore)) {
+                log.warn("CredStore field will be ignored, because value is blank");
```

Comment:
Just to be a bit clearer:
```suggestion
                log.warn("Docker auth config credsStore field will be ignored, because value is blank");
```